### ISSUE TITLE
omni not work for vimtex

### ISSUE THREAD

**Issue by theFool32** [neoclide/coc-sources#1]
Hi, I'm using [vimtex](https://github.com/lervag/vimtex) for latex, it supports omni completion. I wish to use it with coc.nvim.

```
//coc-settings.json
"coc.source.omni.filetypes": ["tex"],
```

```
:echo &omnifunc
vimtex#complete#omnifunc
```

auto-completion fails,  e.g.:
```
\begin{e|}
```
`|` is the cursor.
but `<c-x><c-o>` works.
![image](https://user-images.githubusercontent.com/3063690/50533826-7263e180-0b6d-11e9-8fce-40cf8d28d5fd.png)

BTW, how to use the [pattern](https://github.com/lervag/vimtex/blob/master/autoload/vimtex/re.vim) in coc.nvim

Thanks.

`CocInfo`:
```
## versions

vim version: NVIM v0.3.1
node version: v8.9.0
coc.nvim version: 0.0.42
term: screen-256color
platform: linux

## Error messages
```

**Comment by chemzqm** [comment]
You mean the completion is not triggered?
You can trigger completion manully by things like:
```
  inoremap <silent><expr> <c-space> coc#refresh()
```
Is that pattern used for trigger completion?

**Comment by theFool32** [comment]
`coc#refresh()` not work.

yes, pattern is used for trigger completion.

**Comment by chemzqm** [comment]
Looks like a customized source is needed for support completion from vimex.

**Comment by wbthomason** [comment]
How would a customized source for this work? Would you need to implement `coc#source#vimtex#should_complete(option)` to match against `g:vimtex#re#ncm2` (or similar)?

**Comment by chemzqm** [comment]
It's bug of extract filter text, fixed on 1.2.0.
